specification/cognitiveservices/ContentSafetyStudio/package.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
@suxi-ms this file should be removed and you should rely on the global package.json that is provided at the root of the specs repo.

Reply:
will do